scripts/test/shared.py

```diff
@@ -270,17 +268,7 @@ def has_shell_timeout():
 except (OSError, subprocess.CalledProcessError):
     EMCC = None
 if EMCC is None:
```

Comment:
I think we can remove `EMCC` too, it doesn't look like it's used (and it shouldn't be).

Reply:
Nice!